src/templates.ts

```diff
@@ -87,6 +120,31 @@ export const downloadTemplates =
     }
   }
 
+const enrichWithPrePendingHeader =
+  (mustacheRenderedContent: string, template: Template, codeowners: string) =>
+  (log: Logger): string => {
+    const templateExtension = template.destinationPath.split('.').pop() as string
+    if (!(templateExtension == 'yaml' || templateExtension == 'toml' || templateExtension == 'yml')) {
+      log.debug(`File extension: ${templateExtension} with not supported comments`)
+      return mustacheRenderedContent
+    }
+
+    const codeOwnersEntries = parse(codeowners)
+    const matchedCodeOwner = matchFile(template.sourcePath, codeOwnersEntries)
+
+    const header = process.env.PREPENDING_HEADER_TEMPLATE
+      ? process.env.PREPENDING_HEADER_TEMPLATE
+      : '#OWNER: {{{stewards}}}'
```

Comment:
These are equivalent 😄 

```suggestion
    const header = process.env.PREPENDING_HEADER_TEMPLATE || '#OWNER: {{{stewards}}}'
```